README.md

```diff
@@ -146,19 +142,19 @@ Most services *can* be installed on the Medium cluster, but not all at the same
 
     This will update `/etc/hosts` every time VMs are created or destroyed.
 
-    To avoid entering your password on `vagrant up` & `vagrant destroy` you may enable [passwordless sudo](https://github.com/smdahlen/vagrant-hostmanager#passwordless-sudo).
+    To avoid entering your password on `vagrant up` & `vagrant destroy`, enable [passwordless sudo](https://github.com/smdahlen/vagrant-hostmanager#passwordless-sudo).
 
     On some versions of Mac OS X, installing vagrant plugins may require [installing a modern version of Ruby](./docs/install-ruby.md).
 
 1. Download the DC/OS Installer
 
-    If you don't already have a DC/OS Installer downloaded, you'll need to select and download one of the [supported versions](#supported-dcos-versions).
+    If you don't already have a DC/OS installer downloaded, you'll need to select and download one of the [supported versions](#supported-dcos-versions).
```

Comment:
bleh. fine. they need to rename the whole thing already. no one knows what to call the dcos_generate_config.sh